src/services/orders.ts

```diff
@@ -12,15 +12,27 @@ import { getWeb3Wrapper } from './web3_wrapper';
 
 const logger = getLogger('Services::Orders');
 
-export const getAllOrders = (baseToken: Token, quoteToken: Token) => {
+const getAllOrders = async (baseToken: Token, quoteToken: Token, makerAddresses: string[] | null) => {
```

Comment:
It feels a little weird to have `getUserOrders` and then also `getAllOrders ` which takes in a list of `makerAddresses`. On first pass I would expect you to expand `getUserOrders` to take in multiple user addresses instead -- but I understand why you went about developing it this way